common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -90,6 +90,7 @@ case class WorkData[State <: DataState](
   thumbnail: Option[LocationDeprecated] = None,
   production: List[ProductionEvent[State#MaybeId]] = Nil,
   language: Option[Language] = None,
+  languages: List[Language] = List.empty,
```

Comment:
All the other empty lists here are `Nil` which would be my preference - either way it should be consistent

Reply:
I'd missed that, will fix.